examples/app-setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+export FLASK_APP=app.py
+
+# Install specific dependencies
+pip install -r requirements.txt || exit 1
```

Comment:
What about using [`set -e`](http://www.gnu.org/software/bash/manual/bashref.html#The-Set-Builtin) instead of `|| exit 1`?